nextflow_schema.json

```diff
@@ -167,8 +167,8 @@
                 },
                 "eukulele_db": {
                     "type": "string",
-                    "default": "phylodb",
-                    "description": "Specify which database you would like to use with eukulele, possible alternatives: phylodb, mmetsp. default: phylodb"
+                    "description": "EUKulele database.",
+                    "help_text": "This option allows to use a specific database (or set of databases when separated by commas) with Eukulele. Databases will be downloaded if not already present inside the database directory (see `--eukulele_dbpath`). Possible alternatives: phylodb, mmetsp, marmmetsp, eukprot. NB: you can't use this option with a custom database as eukulele will not recognize the name and it will start to download phylodb by default. If you want to use a custom database, please skip this option and specify only `--eukulele_dbpath`."
```

Comment:
I suggest amending to:
"This option allows the user to specify which database (or set of databases, separated by comma) to use with EUKulele. Databases that are provided with EUKulele will be downloaded if not already present inside the database directory (see `--eukulele_dbpath`). Possible alternatives: phylodb, mmetsp, marmmetsp, eukprot. NB: you can't use this option with a custom database as eukulele will not recognize the name and it will start to download phylodb by default. If you want to use a custom database, please skip this option and specify only `--eukulele_dbpath`."

Reply:
Dito!